src/cc/api/BPF.cc

```diff
@@ -218,6 +218,18 @@ StatusTuple BPF::attach_uprobe(const std::string& binary_path,
                                bpf_probe_attach_type attach_type, pid_t pid,
                                uint64_t symbol_offset) {
 
+  return attach_uprobe(binary_path, symbol, probe_func, symbol_addr, attach_type, pid,
+                symbol_offset, 0);
+}
+
+StatusTuple BPF::attach_uprobe(const std::string& binary_path,
+                               const std::string& symbol,
+                               const std::string& probe_func,
+                               uint64_t symbol_addr,
+                               bpf_probe_attach_type attach_type, pid_t pid,
+                               uint64_t symbol_offset,
```

Comment:
can't use default arguments in this one or it is ambiguous, so to use ref_ctr you have to fully specify all values, even ones with defaults.